docs/book/src/quick-start.md

```diff
@@ -41,17 +41,20 @@ Also, you can install a master snapshot from `https://go.kubebuilder.io/dl/lates
 
 ## Create a Project
 
-Create a directory, and then run the following command inside of it to initialize a new project:
+Create a directory, and then run the init command inside of it to initialize a new project. Follows an example.
 
 ```bash
-kubebuilder init --domain my.domain
+$ mkdir $GOPATH/src/example
+$ cd $GOPATH/src/example
+$ kubebuilder init --domain my.domain
 ```
 
 <aside class="note">
 <h1>Not in $GOPATH</h1>
 
-If you're not in `GOPATH`, you'll need to run `go mod init <modulename>`
-in order to tell kubebuilder and Go the base import path of your module.  
+If you're not in `GOPATH`, you'll need to run `go mod init <modulename>` in order to tell kubebuilder and Go the base import path of your module. 
+
+For a further understand over `GOPATH` see [The GOPATH environment variable][GOPATH-golang-docs] in the [How to Write Go Code][how-to-write-go-code-golang-docs] golang page doc.   
```

Comment:
replace "further understand over " with "further understanding of"

Reply:
Done.